charts/gardener/controlplane/charts/runtime/templates/admission-controller/configmap-componentconfig.yaml

```
@@ -16,8 +16,12 @@ data:
     apiVersion: admissioncontroller.config.gardener.cloud/v1alpha1
     kind: AdmissionControllerConfiguration
     gardenClientConnection:
-      acceptContentTypes: {{ required ".Values.global.admission.config.gardenClientConnection.acceptContentTypes is required" .Values.global.admission.config.gardenClientConnection.acceptContentTypes }}
-      contentType: {{ required ".Values.global.admission.config.gardenClientConnection.contentType is required" .Values.global.admission.config.gardenClientConnection.contentType }}
+      {{- with .Values.global.admission.config.gardenClientConnection.acceptContentTypes }}
+      acceptContentTypes: {{ . | quote }}
+      {{- end }}
```

Nit: Is there a specific reason using `with` instead of `if`? I would vote to use `if` just for consistency reason as currently across the gardener chart `if` is being used in a consistent manner.

```suggestion
      {{- if .Values.global.admission.config.gardenClientConnection.acceptContentTypes }}
      acceptContentTypes: {{ .Values.global.admission.config.gardenClientConnection.acceptContentTypes }}
      {{- end }}
```

---

Hmm, I see a few usages of `with`.
Personally, I prefer `with` (especially for templating a single field), because it allows you to not duplicate the value path.
But I'm open to changing it, if you think consistency is a concern here.